src/mame/nmk/nmk16.cpp

```diff
@@ -3864,32 +3869,33 @@ void nmk16_state::set_hacky_interrupt_timing(machine_config &config)
 void nmk16_state::set_hacky_screen_lowres(machine_config &config)
 {
 	SCREEN(config, m_screen, SCREEN_TYPE_RASTER);
-	//m_screen->set_raw(XTAL(12'000'000)/2, 384, 0, 256, 278, 16, 240); // confirmed
-	m_screen->set_refresh_hz(56.18);
-	m_screen->set_vblank_time(ATTOSECONDS_IN_USEC(3450));
-	m_screen->set_size(256, 256);
-	m_screen->set_visarea(0*8, 32*8-1, 2*8, 30*8-1);
+	m_screen->set_raw(XTAL(12'000'000)/2, 384, 92, 348, 278, 16, 240); // confirmed
+	//m_screen->set_refresh_hz(56.18);
+	//m_screen->set_vblank_time(ATTOSECONDS_IN_USEC(3450));
+	//m_screen->set_size(256, 256);
+	//m_screen->set_visarea(0*8, 32*8-1, 2*8, 30*8-1);
 	m_screen->set_palette(m_palette);
 
 	NMK_16BIT_SPRITE(config, m_spritegen, XTAL(12'000'000)/2);
-	m_spritegen->set_screen_size(384, 256);
+	m_spritegen->set_screen_size(92+348, 16+240);
 	m_spritegen->set_max_sprite_clock(384 * 263); // from hardware manual
+	m_spritegen->set_videoshift(92);
 }
 
 void nmk16_state::set_hacky_screen_hires(machine_config &config)
 {
 	SCREEN(config, m_screen, SCREEN_TYPE_RASTER);
-	//m_screen->set_raw(XTAL(16'000'000)/2, 512, 0, 384, 278, 16, 240); // confirmed
-	m_screen->set_refresh_hz(56.18);
-	m_screen->set_vblank_time(ATTOSECONDS_IN_USEC(3450));
-	m_screen->set_size(512, 256);
-	m_screen->set_visarea(0*8, 48*8-1, 2*8, 30*8-1);
+	m_screen->set_raw(XTAL(16'000'000)/2, 512, 28, 412, 278, 16, 240); // confirmed
+	//m_screen->set_refresh_hz(56.18);
+	//m_screen->set_vblank_time(ATTOSECONDS_IN_USEC(3450));
+	//m_screen->set_size(512, 256);
+	//m_screen->set_visarea(0*8, 48*8-1, 2*8, 30*8-1);
 	m_screen->set_palette(m_palette);
 
 	NMK_16BIT_SPRITE(config, m_spritegen, XTAL(16'000'000)/2);
-	m_spritegen->set_screen_size(384, 256);
-	m_spritegen->set_max_sprite_clock(512 * 263); // not verified?
-	m_spritegen->set_videoshift(64);
+	m_spritegen->set_screen_size(28+412, 16+240);
+	m_spritegen->set_max_sprite_clock(384 * 263); // not verified?
+	m_spritegen->set_videoshift(28+64);
 }
```

Comment:
You don’t need to leave the commented non-raw screen parameters if you know the real values.

Reply:
OK, I will remove it.